### PR TITLE
fix: adds missing allowFailureMap parameter to IDAO.Executed event

### PR DESCRIPTION
**Pull request by mathewmeconry**
## Description

Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)

Task: [ID]()

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: packages/contracts/src/core/dao/IDAO.sol [hunk 1]
```diff
@@ -55,13 +55,15 @@ interface IDAO {
     /// @param actions The array of actions executed.
     /// @param failureMap The failure map encoding which actions have failed.
     /// @param execResults The array with the results of the executed actions.
+    /// @param allowFailureMap The encoded uint to mark which actions are allowed to fail.
```
Comment (Michael-A-Heuer) [inline packages/contracts/src/core/dao/IDAO.sol:58]:
```suggestion
    /// @param failureMap The allow failure map encoding which actions are allowed to fail.
```
Is it possible to rearrange the order?

```solidity
    event Executed(
        address indexed actor,
        bytes32 callId,
        Action[] actions,
        uint256 allowFailureMap,
        uint256 failureMap,
        bytes[] execResults
    );
```
or 

```solidity
    event Executed(
        address indexed actor,
        bytes32 callId,
        Action[] actions,
        uint256 allowFailureMap,
        bytes[] execResults,
        uint256 failureMap
    );
```
seems more logical to me. (`allowFailureMap` as an input coming before `failureMap`, the output)

File: packages/contracts/src/core/dao/IDAO.sol [hunk 2]
```diff
@@ -55,13 +55,15 @@ interface IDAO {
...
     /// @dev The value of `callId` is defined by the component/contract calling the execute function. A `Plugin` implementation can use it, for example, as a nonce.
     event Executed(
         address indexed actor,
         bytes32 callId,
         Action[] actions,
         uint256 failureMap,
-        bytes[] execResults
+        bytes[] execResults,
+        uint256 allowFailureMap
```
Comment (Michael-A-Heuer) [inline packages/contracts/src/core/dao/IDAO.sol:66]:
```suggestion
    /// @param failureMap The allow failure map encoding which actions are allowed to fail.
```
Is it possible to rearrange the order?

```solidity
    event Executed(
        address indexed actor,
        bytes32 callId,
        Action[] actions,
        uint256 allowFailureMap,
        uint256 failureMap,
        bytes[] execResults
    );
```
or 

```solidity
    event Executed(
        address indexed actor,
        bytes32 callId,
        Action[] actions,
        uint256 allowFailureMap,
        bytes[] execResults,
        uint256 failureMap
    );
```
seems more logical to me. (`allowFailureMap` as an input coming before `failureMap`, the output)

**Review by Michael-A-Heuer — CHANGES_REQUESTED** [review]

File: packages/subgraph/src/dao/dao_v1_2.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  proposalId = proposalId
+    .concat('_')
+    .concat(event.transaction.hash.toHexString())
+    .concat('_')
+    .concat(event.transactionLogIndex.toHexString());
+  let proposal = new TransactionActionsProposal(proposalId);
+  proposal.dao = event.address.toHexString();
+  proposal.createdAt = event.block.timestamp;
+  proposal.endDate = event.block.timestamp;
+  proposal.startDate = event.block.timestamp;
```
Comment (Michael-A-Heuer) [inline packages/subgraph/src/dao/dao_v1_2.ts:23]: Is `event.block.timestamp;` for all three fields intentional?

File: packages/subgraph/src/dao/dao_v1.ts [hunk 4]
```diff
@@ -11,7 +11,7 @@ import {
   SignatureValidatorSet,
   StandardCallbackRegistered,
   CallbackReceived
-} from '../../generated/templates/DaoTemplate/DAO';
+} from '../../generated/templates/DaoTemplateV1/DAO';
```
Comment (Michael-A-Heuer) [inline packages/subgraph/src/dao/dao_v1.ts:14]: Is the versioning number following the @aragon/osx npm number?
Should we use 
V1_0 and V1_2  or 
V1_0_0 and V1_2_0 ?

**Review by Michael-A-Heuer — COMMENTED** [review]
Some questions / suggestions

File: packages/subgraph/src/dao/dao_v1.ts [hunk 5]
```diff
@@ -69,6 +69,8 @@ export function handleExecuted(event: Executed): void {
     .concat('_')
     .concat(event.params.callId.toHexString());
 
+  event.params.callId;
+
```
[the remark this answers is not in the document]
Comment (mathewmeconry) [reply packages/subgraph/src/dao/dao_v1.ts:73]: Removed

File: packages/subgraph/src/dao/dao_v1.ts (hunk 4 above)
Comment (mathewmeconry) [reply packages/subgraph/src/dao/dao_v1.ts:14]: good point. Done

**Review by Michael-A-Heuer — APPROVED** [review]
LGTM

**Review by Rekard0 — APPROVED** [review]
LGTM!

**Pull request merged**